components/src/core/Drawer/DrawerNavItem/DrawerNavItem.tsx

```diff
@@ -153,13 +154,13 @@ const InfoListItemRoot = styled(InfoListItem, {
         '& .BluiInfoListItem-subtitle': {
             opacity: hidePadding && !icon && drawerOpen ? 1 : hidePadding && !icon ? 0 : 'inherit',
             transition: hidePadding && !icon ? theme.transitions.create('opacity') : '',
-            color: !active && theme.palette.mode === 'dark' ? theme.palette.text.secondary : undefined,
+            color: !active && theme.applyStyles('dark', { color: theme.vars.palette.text.secondary }),
```

Comment:
This is a syntax I have not seen before...does this work? theme.applyStyles returns an object, so my intuition says this gets compiled into something like:
```
color: {
    color: 'variable-name'
}
```
which doesn't make sense.

I would expect this to look more like:
```
'& .BluiInfoListItem-subtitle':{
    opacity: value,
    transition: value,
    ...theme.applyStyles('dark', {
        color: !active && (theme.vars || theme).palette.text.secondary
    })
}
```